.github/workflows/ci.yml

```diff
@@ -126,15 +126,31 @@ jobs:
             v1-go-${{ runner.os }}
       - name: Set up Go
         uses: actions/setup-go@v3
+      - name: Build go
+        working-directory: go-grpc-gateway
+        run: ./install_tools.sh && ./build.sh
+      - name: Check dirty git
+        uses: ./.github/actions/check-dirty-git
```

Comment:
do we need to check after each step or is at the end sufficient?
```suggestion
```

Reply:
idk, the original ci had this at both steps, it seems like it doesn't hurt to do it multiple times

Reply:
I agree it's only calling git status. Linux with few if any submodules should be a sub-second operation. 
(Windows with submodules == :puke:)